vrx_gazebo/worlds/sydneyregatta.xacro

```diff
@@ -11,15 +11,15 @@
       <grid>0</grid>
       <origin_visual>0</origin_visual>
     </scene>
-    <!-- Estimated latitude/longitude of sydneyregatta from satellite imagery -->
+    <!-- Estimated latitude/longitude of sydneyregatta
+	 from satellite imagery -->
     <spherical_coordinates>
       <surface_model>EARTH_WGS84</surface_model>
       <world_frame_orientation>ENU</world_frame_orientation>
       <latitude_deg>-33.724223</latitude_deg>
       <longitude_deg>150.679736</longitude_deg>
       <elevation>0.0</elevation>
-      <!-- For legacy gazebo reasons, need to rotate -->
-      <heading_deg>180</heading_deg>
```

Comment:
This was done to have the Gazebo x/y and Hector Lat/Lon be consistent.   I'm not sure if this has other effects?

Reply:
As @caguero mentions in PR #341 - we need to be careful with `referenceHeading`, so would appreciate a double check.